actor-util/src/main/java/org/sunbird/actorutil/org/OrganisationClient.java

```diff
@@ -58,4 +59,11 @@
    */
   List<Organisation> esSearchOrgByFilter(Map<String, Object> filter);
 
+  /**
+   * Search organisations using specified searchDto.
+   *
+   * @param searchDto SearchDTO search criteria for organisation search
+   * @return List of organisations
+   */
+  List<Organisation> esSearchOrgBySearchDto(SearchDTO searchDto);
```

Comment:
client should not expose very generic apis, it should have specific parameters and names. which is easier to consume from outside